File: co.edu.uniandes.fixml.toC++/test/text2.xml/Amt.cpp
#include <iostream> 

using namespace std;

class Amt {

	private:
  	string Typ_;
  	string Amt_;
 	
 	public: 

  	// default constructor
	Amt () {
		Typ_ = "FMTM";
		Amt_ = "0.00";
  	
  	}
  
  	// by parameters
  	Amt (		string Typ_p, string Amt_p){
	  
		Typ_ = Typ_p;
		Amt_ = Amt_p;
	  
  	}
};

File: co.edu.uniandes.fixml.toCSharp/test/text2.xml/Amt.cs


// Class for tag Amt
class Amt {

	string _Typ;
	string _Amt;
	
	
	// default constructor
	
	public Amt ( ) {   
		this._Typ = "FMTM";
		this._Amt = "0.00";
		
		
	}
	
	public Amt ( string Typ, string Amt ) {
		this._Typ = Typ;
		this._Amt = Amt;
		
	}
	
	
} 